### PR TITLE
Fixup review

### PR DESCRIPTION
**Pull request by seberm**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,10 +1,10 @@
...
-.repos
+\.pytest_cache
+\.repos
 venv
-.venv
-.idea
-.vscode
+\.venv
+\.idea
+\.vscode
```
Comment (LecrisUT) [inline .gitignore:6]: Why the `\`? Shouldn't it be `/`?

File: pyproject.toml [hunk 2]
```diff
@@ -24,8 +24,8 @@ classifiers = [
   "Operating System :: POSIX :: Linux",
 ]
 dependencies = [
-  "tmt~=1.35",
-  "fastapi~=0.112",
+  "tmt~=1.36",
+  "fastapi~=0.115",
```
Comment (LecrisUT) [inline pyproject.toml:28]: Not much worth locking dependencies, I think would be better to only `>=` if even necessary, github actions would pick up any incompatibilities, and probably there will be some packit packaging for the rest.

File: src/tmt_web/generators/html_generator.py [hunk 3]
```diff
@@ -4,14 +4,16 @@
 from jinja2 import Environment, FileSystemLoader
 from tmt import Logger, Plan, Test
 
-templ_dir = Path(__file__).resolve().parent / 'templates'
+templ_dir = Path(__file__).resolve().parent / "templates"
```
Comment (LecrisUT) [inline src/tmt_web/generators/html_generator.py:7]: There is a `ruff` setting for that. https://docs.astral.sh/ruff/settings/#lint_flake8-quotes_docstring-quotes

File: src/tmt_web/generators/json_generator.py [hunk 4]
```diff
@@ -7,7 +7,7 @@
 
 def _create_json_data(obj: Test | Plan, logger: tmt.Logger) -> dict[str, Any]:
     """
-    Helper function to create the JSON data from a test or plan object
+    Helper function to create the JSON data from a test or plan object.
```
Comment (LecrisUT) [inline src/tmt_web/generators/json_generator.py:10]: Should enable `D` ruff rules for these.

File: .gitignore (hunk 1 above)
Comment (seberm) [reply .gitignore:6]: Oh, sorry. My bad. I confused an expression with a regex and wanted to make sure the `.` is there :man_facepalming:.  Let's keep it as is.

Comment (LecrisUT) [reply .gitignore:6]: Yeah, glob vs regex always gets me as well. What's worse gitignore syntax is more than just glob :grimacing:

File: pyproject.toml (hunk 2 above)
Comment (martinhoyer) [reply pyproject.toml:28]: hmm, probably not super important, but I'd keep it as is, unless you mean just specific packages. fwiw, they are not locked, well, only the major version is.

File: src/tmt_web/generators/html_generator.py (hunk 3 above)
Comment (martinhoyer) [reply src/tmt_web/generators/html_generator.py:7]: That's probably a comment to the #4 or separate PR entirely imho.

File: src/tmt_web/generators/json_generator.py (hunk 4 above)
Comment (martinhoyer) [reply src/tmt_web/generators/json_generator.py:10]: That's probably a comment to the https://github.com/teemtee/web/pull/4 or separate PR entirely imho.

File: src/tmt_web/generators/html_generator.py (hunk 3 above)
Comment (psss) [reply src/tmt_web/generators/html_generator.py:7]: Ack, let's resolve this separately. I don't think we have any preference about quotes.

**Review by psss — APPROVED** [review]
Overall changes seem reasonable, let's merge this to finish the original pull request soon. Thanks for the cleanup!

**Pull request merged**